CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 - add show_app_in_chat option to webxdc info message context menu #4459
 - add experimental content protection option (to prevent screenshots and screenrecording the app) #4475
 - app picker for webxdc apps in attachement menu #4485
+- add special error dialog for the case that deltachat-rpc-server is not found #4479
```

Comment:
needs to be rebased before merging, so that the changelog entry is not in the past/already-released version.